centrallix/objectsystem/obj_datatypes.c

```diff
@@ -5,6 +5,7 @@
 #include <time.h>
 #include <stdlib.h>
 #include <math.h>
+#include <byteswap.h>
```

Comment:
confirm availability

Reply:
@gbeeley As far as I can tell, `byteswap.h`/`bswap_64()` are part of the GNU C extensions. Is this safe to use without checking for its availability, or should I do some checking and conditional compilation like I did in #17?

Reply:
Currently Centrallix uses the GNU C toolchain throughout.  We haven't done any work on building using LLVM or whatnot, and indeed ./configure checks for __GNUC__.  So, yes, it is acceptable to use GNU C extensions without preprocessor directives.